.tekton/vmaas-push.yaml

```diff
@@ -28,6 +28,12 @@ spec:
     value: Dockerfile
   - name: path-context
     value: .
+  - name: hermetic
+    value: "true"
+  - name: prefetch-input
```

Comment:
I see that prefetching task is set for python and rpm dependencies, but what about golang dependencies?

https://github.com/RedHatInsights/vmaas/blob/master/vmaas-go/go.sum

Reply:
Pre-fetching of go packages should be also possible, added it here https://github.com/RedHatInsights/vmaas/pull/1436/commits/b9933ca6b9270b239f728446a812ca855c2eb15d, let's see if it is working...